api.R

```diff
@@ -11,91 +12,130 @@ source("generics.R")
 # Read AWS creds from Docker secrets
 if (file.exists("/run/secrets/ENV_FILE")) {
   readRenviron("/run/secrets/ENV_FILE")
-} else {
+} else if (file.exists("secrets/ENV_FILE")) {
   readRenviron("secrets/ENV_FILE")
+} else {
+  readRenviron(".env")
 }
-readRenviron(".env")
```

Comment:
The logic here was a bit confusing for local development, where neither `/run/secrets/ENV_FILE` nor `secrets/ENV_FILE` exist. I think the new conditional structure should make local development easier, but let me know if I'm misinterpreting something.

Reply:
That makes sense, I refactored in abe540391c05b750507e5086fedbf27b31c49d51 to always load this file and only load `secrets/ENV_FILE` if it exists (similar to `/run/secrets/ENV_FILE`).